apps/dashboard/src/index.css

```diff
@@ -18,7 +35,27 @@
     --foreground-900: 226 21% 12%;
     --foreground-950: 222 32% 8%;
 
-    --primary: 346 73% 50%;
+    --novu-800: 346 72% 42%;
```

Comment:
Is Novu color the primary color?

Reply:
Align doesa separation between the color pallete, and their semantic usage. Novu is the color palette, while primary is the semantic usage of the novu color.